### ISSUE TITLE
Couldn't able to open

### ISSUE THREAD

**Issue by nkalapala12** [hiaaryan/wora#32]
It's saying the developer can't be verified

<img width="261" alt="Screenshot 2024-09-08 at 6 28 16 PM" src="https://github.com/user-attachments/assets/28ffc3d7-7886-46fe-a7df-036a651ca459">

How can it be resolved? @hiaaryan

**Comment by Tmpered** [comment]
Right-click>Open, or go to Settings>Privacy And Security. Scroll down and press "Open" or "Allow" and input your password, should be fine from then

**Comment by nkalapala12** [comment]
Sure, Let me try this , Thank you.
